src/test/java/uk/gov/justice/digital/delius/integration/secure/AttendanceResourceAPITest.java

```diff
@@ -81,19 +78,21 @@ public void getBadRequest400() {
     }
 
     @Test
-    public void getKnownCrnButEventIdNotFound404() {
+    public void getKnownCrnButEventIdNotFound200() {
```

Comment:
I think really it should be
1) Offender not found 404
2) Event not found 404
3) No attendances found 200 with empty list

However, given this isn't a big deal  lets just get this merged and address it later - it is not likely to cause a real world issue for the courts project